client/lib/js/react/components/account_detail.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import classnames from 'classnames';
+import { Row } from 'react-bootstrap/lib';
+
+
+export default function AccountDetail(props) {
```

Comment:
Not sure if it's worth it, but this could probably be generalized so the Interledger version could wrap around this.